wallet/src/main/scala/org/bitcoins/wallet/models/IncomingTransactionTable.scala

```diff
@@ -0,0 +1,134 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.crypto._
+import org.bitcoins.core.number.UInt32
+import org.bitcoins.core.protocol.script.EmptyScriptSignature
+import org.bitcoins.core.protocol.transaction.{
+  BaseTransaction,
+  EmptyWitness,
+  Transaction,
+  TransactionInput,
+  WitnessTransaction
+}
+import slick.jdbc.SQLiteProfile.api._
+import slick.lifted.{PrimaryKey, ProvenShape}
+
+/** Represents a relevant transaction for the wallet that we should be keeping track of */
+case class IncomingTransactionDb(
+    txIdBE: DoubleSha256DigestBE,
+    transaction: Transaction,
+    unsignedTxIdBE: DoubleSha256DigestBE,
+    unsignedTx: Transaction,
```

Comment:
The scaladoc isn't enough for me to know what the difference between `transaction` and `unsignedTx`, why both?